### ISSUE TITLE
CodeGPT suppresses the new Jetbrains feature for local Full Line Code Completion, even if it does not work with current configuration

### ISSUE THREAD

**Issue by niceapps-ch** [carlrobertoh/CodeGPT#448]
### What happened?

CodeGPT does not support code completion when connected to an OpenAI Service or local via Ollama or LM Studio.
JetBrains introduced an integrated full line code completion feature that uses a very small local model. This new feature does not work when CodeGPT is active, even if CodeGPT does not do any code completion with the active configuration.

Proposal:
- Make code completion configurable, so I can turn it off to use the native feature of IntelliJ and still use all the rest of CodeGPT.

### CodeGPT version

2.6.0

### Operating System

macOS

**Comment by boswelja** [comment]
Mildly unrelated, but how well does the built-in code completion work? I use Android Studio, so I won't get to use it for a little bit

**Comment by carlrobertoh** [comment]
Thanks for reporting, I'll take a look. 
This should only happen in IDE versions starting from 2024.1.

**Comment by niceapps-ch** [comment]
> Mildly unrelated, but how well does the built-in code completion work? I use Android Studio, so I won't get to use it for a little bit

I'm not sure, didn't get to try it productively yet. And probably won't until this is fixed, because I'm too happy with CodeGPT :)

**Comment by reneleonhardt** [comment]
The bug was introduced in 2.6.0 https://github.com/carlrobertoh/CodeGPT/commit/f0172722c75ae50d2ea895f68cbef0c90bbbcc7f#diff-79d11c7104a03e0a8b9c00e9e8cc953346a515a4feeb91fb98334122664f4a73R47
I will provide a fix later and write a test when there is time, pleasy try again with the next release.

**Comment by carlrobertoh** [comment]
Most likely, CodeGPT's inline suggestions provider is interfering with IntelliJ's one, altho I'm not 100% sure.
https://github.com/carlrobertoh/CodeGPT/blob/master/src/main/resources/META-INF/plugin.xml#L43

**Comment by reneleonhardt** [comment]
@niceapps-ch Can you try again after disabling the plugin `Full line code completion` or disabling the setting?
https://blog.jetbrains.com/blog/2024/04/04/full-line-code-completion-in-jetbrains-ides-all-you-need-to-know/#how-to-tweak-the-feature

**Comment by carlrobertoh** [comment]
@niceapps-ch which IDE are you using, and how did you confirm that the native completions are suppressed? 
I'm unable to reproduce this issue, for me it behaves correctly - when the CodeGPT completions are turned off, I receive the native ones, and vice versa.

WebStorm 2024.1

**Comment by niceapps-ch** [comment]
@carlrobertoh @reneleonhardt Thank you guys for looking into it! I'm using mainly Webstorm.

You are absolutely right. Now it seems to be no issue anymore. Yesterday it only worked when I disabled the CodeGPT plugin and didn't work again after I enabled CodeGPT Plugin, but maybe something else was going on on my machine. 

I have only the Custom OpenAI setting configured and don't use any other configurations on this machine.
Using mainly webstorm by the way.

Will close the issue. Thanks again.
